tutorials/vr/vr_primer.rst

```diff
@@ -32,52 +45,134 @@ To enable an interface, you execute the following code:
+where it is, the position of the camera and controllers will be adjusted according to your
+movements. When you move through the virtual world, either through controller input or when
+you implement a teleport system, it is the position of the origin point which you will
+have to adjust.
+
+:ref:`ARVRCamera <class_ARVRCamera>` is the second node that must always be a part of your
+scene and it must always be a child node of your origin node. It is a subclass of Godot's
+normal camera. However, its position is automatically updated each frame based on the physical
+orientation and position of the HMD. Also due to the precision required for rendering to an
+HMD or rendering an AR overlay over a real world camera, most of the standard camera properties
+are ignored. The only properties of the camera that are used are the near and far plane
+settings. The FOV, aspect ratio and projection mode are all ignored.
+
+Note that, for our native mobile VR implementation, there is no positional tracking, only
+the orientation of the phone and by extension, the HMD is tracked. This implementation
+artificially places the camera at a height (Y) of 1.85.
 
 Conclusion: your minimum setup in your scene to make AR or VR work should look like this:
 
 .. image:: img/minimum_setup.png
 
-And that's all you need to get started. Obviously, you need to add something more into your scene, so there is something to see, but after that, you can export the game to your phone of choice, pop it into a viewer and away you go.
+And that's all you need to get started with the native mobile interface. Obviously, you need
+to add something more into your scene, so there is something to see, but after that, you can
+export the game to your phone of choice, pop it into a viewer and away you go.
 
-Other things to consider
-------------------------
+Official plugins and resources
+------------------------------
+
+As mentioned earlier, Godot does not support the various VR and AR SDKs out of the box, you
+need a plugin for the specific SDK you want to use. There are several official plugins available
+in the `GodotVR Repository <https://github.com/GodotVR>`_.
+
+* `Godot Oculus Mobile <https://github.com/GodotVR/godot_oculus_mobile>`_ provides support for
+  the Oculus Go and Oculus Quest. The Quest will require additional setup documented `here <doc_developing_for_oculus_quest>`_.
+* `Godot OpenVR <https://github.com/GodotVR/godot_openvr>`_ (not to be confused with OpenXR)
+  supports the OpenVR SDK used by Steam.
+* `Godot Oculus <https://github.com/GodotVR/godot_oculus>`_ supports the Oculus SDK
+  (desktop headsets only).
+* `Godot OpenHMD <https://github.com/GodotVR/godot_openhmd>`_ supports OpenHMD, an open source
+  API and drivers for headsets.
 
-There are a few other subjects that we need to briefly touch upon in this primer that are important to know.
+These plugins can be downloaded from github or the Godot Asset Library. The author of them will
```

Comment:
"The author of them" sounds awkward, I'd use "Their author".

But I'd also prefer to simply drop the sentence, I don't think it's relevant information for the docs.